Segments/Blackburrow.mapproj

```diff
@@ -32502,24 +32502,20 @@ return snake;
     IceProtection = 100,
     Immunity = CreatureImmunity.Poison,
     Experience = 25000,
-    
-    VisibilityDistance = 0,
+    HideDetection = 50,
+    VisibilityDistance = 1,
 };
 
 elemental.Attacks = new CreatureAttackCollection
 {
 
-    new CreatureBasicAttack(15),
+    {
+        new CreatureAttackEntry(12, 10, 20, "You feel poison burn through your veins as the arrow strikes", new AttackPoisonComponent(15))
+    }
 };
 
-elemental.Spells = new CreatureSpellCollection()
-    {
-       { new CreatureSpell<PoisonCloudSpell>(
-            skillLevel: 11, cost: 16, 
-            mantra: SpellHelper.GenerateMantra(), instantCast: false),   100 },
-    };
 
-elemental.Wield(new GiltDagger());
+elemental.Wield(new FineCrossbow());
```

Comment:
Giving this guy ranged and 1 visibility means he's going to run from you when you're close enough to see him. The only way to kill him is going to be to aoe the area.

Reply:
Good call, i'm dumb... updated.